prometheus_client/openmetrics/exposition.py

```diff
@@ -20,34 +30,40 @@ def _is_valid_exemplar_metric(metric, sample):
     return False
 
 
-def generate_latest(registry):
+def generate_latest_fn(escaping):
```

Comment:
I think this can be removed an replaced with `functools.partial` where called. If not, I think at least prefixing with `_` would be nice.

Reply:
didn't know about that one!